modules/tinymce/src/plugins/image/test/ts/browser/plugin/MainTabTest.ts

```diff
@@ -1,58 +1,50 @@
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/image/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('All image dialog ui options on empty editor' + (Env.ceFalse ? '' : ' (old IE)'), (success, failure) => {
-  SilverTheme();
-  ImagePlugin();
-  Pipeline.async({}, [
-    Log.chainsAsStep('TBA', 'Image: all image dialog ui options on empty editor' + (Env.ceFalse ? '' : ' (old IE)'), [
-      Editor.cFromSettings({
-        ...silverSettings,
-        image_caption: true,
-        image_list: [
-          { title: 'link1', value: 'link1' },
-          { title: 'link2', value: 'link2' }
-        ],
-        image_class_list: [
-          { title: 'None', value: '' },
-          { title: 'class1', value: 'class1' },
-          { title: 'class2', value: 'class2' }
-        ]
-      }),
-      cExecCommand('mceImage', true),
-      cWaitForDialog(),
-      cFillActiveDialog({
-        src: { value: 'src' },
-        alt: 'alt',
-        class: 'class1',
-        dimensions: {
-          width: '100',
-          height: '200'
-        },
-        caption: true
-      }),
-      cSubmitDialog(),
-      cAssertCleanHtml('Checking output', (() => {
-        if (Env.ceFalse) {
-          return (
-            '<figure class="image">' +
-            '<img class="class1" src="src" alt="alt" width="100" height="200" />' +
-            '<figcaption>Caption</figcaption>' +
-            '</figure>'
-          );
-        } else { // old IE
-          return (
-            '<p><img class="class1" src="src" alt="alt" width="100" height="200" /></p>'
-          );
-        }
-      })()),
-      Editor.cRemove
-    ])
-  ], () => success(), failure);
+import { assertCleanHtml, fillActiveDialog, pWaitForDialog, submitDialog } from '../../module/Helpers';
+
+describe('browser.tinymce.plugins.image.plugin.MainTabTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'image',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+    image_caption: true,
+    image_list: [
+      { title: 'link1', value: 'link1' },
+      { title: 'link2', value: 'link2' }
+    ],
+    image_class_list: [
+      { title: 'None', value: '' },
+      { title: 'class1', value: 'class1' },
+      { title: 'class2', value: 'class2' }
+    ]
+  }, [ Plugin, Theme ]);
+
+  it('TBA: all image dialog ui options on empty editor', async () => {
+    const editor = hook.editor();
+    editor.execCommand('mceImage');
+    await pWaitForDialog(editor);
+
+    fillActiveDialog({
+      src: { value: 'src' },
+      alt: 'alt',
+      class: 'class1',
+      dimensions: {
+        width: '100',
+        height: '200'
+      },
+      caption: true
+    });
+    submitDialog(editor);
+    assertCleanHtml('Checking output', editor, (
+      '<figure class="image">' +
```

Comment:
We no longer need the IE thing?

Reply:
IE thing? Do you mean the `Env.ceFalse` check? If so no, all supported browsers support Content Editable now and if you look at that, it's hard coded to `true.

Reply:
Awesome. The comment said `old IE` but I wasn't sure if that meant "an old version of IE" or "IE 11, but the browser is so old and behind"

Reply:
Yeah, `old IE` here is likely referring to IE 8 or lower...